### ISSUE TITLE
Animator component always plays animation automatically

### ISSUE THREAD

**Issue by eobet** [Unity-Technologies/SimpleAnimation#33]
The SimpleAnimation scrips adds an Animator component to the object, which always plays the animation automatically, _even if you remove the SimpleAnimation script!_

The current "Play automatically" option on the SimpleAnimation script apparently can't override this.

Attaching a project example that confirms this.

[TestProject.zip](https://github.com/Unity-Technologies/SimpleAnimation/files/1846842/TestProject.zip)

**Comment by eobet** [comment]
If you add a new state to an animation controller, set that to the default state and speed to 0, then add that animation controller to the objects and remove the SimpleAnimation, the animation doesn't play automatically anymore.

But this page explicitly states that you can forget about the Animator component:

https://blogs.unity3d.com/2017/11/28/introducing-the-simple-animation-component/

**Comment by tklee1975** [comment]
I changed the 'SimpleAnimation_impl.cs' to fix

```
 protected virtual void OnEnable()
    {
        Initialize();
        m_Graph.Play();
        if (m_PlayAutomatically)
        {
            Stop();
            Play();
        }
    }

```
to 
```
 protected virtual void OnEnable()
    {
        Initialize();
        m_Graph.Play();
        Stop();
           
        if (m_PlayAutomatically)
        {
            Play();
        }
    }
```

**Comment by davidg-unity** [comment]
This issue has been fixed in pull request #42.
